physx-sys/src/physx_api.cpp

```diff
@@ -52,45 +52,73 @@ PxFilterFlags FilterShaderTrampoline(PxFilterObjectAttributes attributes0,
     return PxFilterFlags{shaderfilter(&info)};
 }
 
-class CollisionFilterTrampoline : public PxSimulationEventCallback
+typedef void (*CollisionCallback)(void *, PxContactPairHeader const *, PxContactPair const *, PxU32);
+typedef void (*TriggerCallback)(void *, PxTriggerPair const *, PxU32);
+typedef void (*ConstraintBreakCallback)(void *, PxConstraintInfo const *, PxU32);
+typedef void (*WakeSleepCallback)(void *, PxActor **const, PxU32, bool);
+typedef void (*AdvanceCallback)(void *, const PxRigidBody *const *, const PxTransform *const, PxU32);
```

Comment:
I'd argue these are much more readable as using declarations!

```c++
using CollisionCallback = void(*)(void*, PxContactPairHeader const *, PxContactPair const *, PxU32);
...
```

Reply:
I guess it depends on what you're used to, but fine by me :)